src/test/ObjPropsDynamicSetter.Test.Unit/Models/TestClass.cs

```diff
@@ -9,7 +9,7 @@ public class TestClass
 #pragma warning disable CA1051 // Do not declare visible instance fields
 #pragma warning disable IDE1006 // Naming Styles
 #pragma warning disable SA1401 // Fields should be private
-        public object TestField = 1;
+        public object TestField;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/Molnix888/obj-props-dynamic-setter/pullRequest?prid=6358389)